src/roadveh_cmd.cpp

```diff
@@ -833,10 +841,10 @@ static void RoadVehCheckOvertake(RoadVehicle *v, RoadVehicle *u)
 	 *  - No barred levelcrossing
 	 *  - No other vehicles in the way
 	 */
+
 	od.tile = v->tile;
 	if (CheckRoadBlockedForOvertaking(&od)) return;
-
-	od.tile = v->tile + TileOffsByDiagDir(DirToDiagDir(v->direction));
+	od.tile += TileOffsByDiagDir(DirToDiagDir(v->direction));
```

Comment:
I suspect this is wrong. We use TileOffsByDiagDir() to get the next tile in the correct direction...just incrementing to the next tile index is probably in the wrong direction. (I could be wrong if I'm misreading your usage.)

Reply:
This isn't a behaviour change, od.tile is already initialised v->tile, this change just replaces the reassignment.

Reply:
Aha, I was misreading it. 🙂